community/examples/AMD/hpc-amd-slurm-v6.yaml

```diff
@@ -0,0 +1,228 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: hpc-amd-slurm-v6
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: amd-v6
+  region: us-east4
+  zone: us-east4-c
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network
+    source: modules/network/vpc
+
+  - id: homefs
+    source: modules/file-system/filestore
+    use: [network]
+    settings:
+      local_mount: /home
+
+  - id: swfs
+    source: modules/file-system/filestore
+    use: [network]
+    settings:
+      local_mount: /sw
+
+  - id: spack-setup
```

Comment:
Please insert an explicit spack_ref setting. This will make the blueprint less likely to fail as we update the modules. If you use the default value `v0.20.0`, don't bother re-testing.

Reply:
Yes, I tested this with default one.